quickwit-cli/src/cli.yaml

```diff
@@ -204,8 +204,42 @@ subcommands:
         settings:
             - ArgRequiredElseHelp
         display_order: 2
-        about: Operations (add, delete, describe...) on splits.
+        about: Operations (list, add, delete, describe...) on splits.
         subcommands:
+            - list:
+                about: List all splits of an index.
+                args:
+                    - metastore-uri:
+                        about: Location of the metastore
+                        long: metastore-uri
+                        value_name: METASTORE URI
+                        required: true
+                        env: QUICKWIT_METASTORE_URI
+                    - index-id:
+                        about: Index ID
+                        long: index-id
+                        value_name: INDEX ID
+                        required: true
+                    - states:
+                        about: Comma-separated list of SplitState to filter on.
+                        long: states
+                        value_name: SPLIT STATES
+                        multiple_occurrences: true
+                        use_delimiter: true
+                    - from:
+                        about: Start date to filter on.
+                        long: from
+                        value_name: START TIMESTAMP
+                    - to:
+                        about: End date to filter on.
+                        long: to
+                        value_name: END TIMESTAMP
+                    - tags:
+                        about: Comma-separated list of tags to filter on.
```

Comment:
Is this a disjunction (OR) or a conjunction (AND) ?
I think we want conjunction but the code is a disjunction. Making the doc explicit would be great here.

For instance, "Only splits that contain all of the tags will be returned."